src/net/sourceforge/plantuml/AbstractPSystem.java

```diff
@@ -136,7 +136,7 @@ final public long seed() {
 		return getSource().seed();
 	}
 
-	public int getNbImages() {
+	public int getNbImages(FileFormatOption formatOption) {
```

Comment:
Hi @liudongmiao,

A simple dummy question:
- Why `getNbImages` needs now of `formatOption`?

Regards,
Th.

Reply:
I agree with @The-Lum : this is somehow a breaking change, and does not sound very logical to me.

Could you tell us more about this?

Thanks!

Reply:
In AthomMath, the dimension is cached.

https://github.com/plantuml/plantuml/blob/f186e7a6864933629014681345017e6f42ddce8e/src/net/sourceforge/plantuml/klimt/creole/atom/AtomMath.java#L74-L79

There are two way to fix:
- pass the real `StringBounder` when possible (previous commit)
- don't cache for debug (I have reverted the commit, and make a new commit)